### PR TITLE
Fixes #2

### LINKED ISSUE #2: Feature Request: Override onAuthenticationHelp to allow for actions on "finger moved too fast"
The Google Pixel now supports "swipe for notifications panel" on the fingerprint reader. Although Fingerprint2Sleep can do something similar, it currently only works on successful fingerprints. I wanted to add the ability to catch "finger moved too fast" which is what usually occurs when swiping your finger over the reader.

### PR DESCRIPTION
**Pull request by gerges-zz**
@ztc1997 could you add/provide the correct Chinese translation?

### CODE REVIEW

**Review by Gr307496 — APPROVED** [review]

File: app/src/main/kotlin/com/ztc1997/fingerprint2sleep/FPQAService.kt [hunk 1]
```diff
@@ -8,6 +8,7 @@ import android.hardware.fingerprint.FingerprintManager
 import android.os.CancellationSignal
```
Comment (Eutiago) [inline app/src/main/kotlin/com/ztc1997/fingerprint2sleep/FPQAService.kt:8]: 8

**Review by Eutiago — APPROVED** [review]

File: app/src/main/kotlin/com/ztc1997/fingerprint2sleep/FPQAService.kt [hunk 2]
```diff
@@ -8,6 +8,7 @@ import android.hardware.fingerprint.FingerprintManager
...
 import android.os.IBinder
```
Comment (Eutiago) [inline app/src/main/kotlin/com/ztc1997/fingerprint2sleep/FPQAService.kt:9]: 9

File: app/src/main/kotlin/com/ztc1997/fingerprint2sleep/FPQAService.kt [hunk 3]
```diff
@@ -8,6 +8,7 @@ import android.hardware.fingerprint.FingerprintManager
...
 import android.support.v7.app.NotificationCompat
```
Comment (Eutiago) [inline app/src/main/kotlin/com/ztc1997/fingerprint2sleep/FPQAService.kt:10]: 10

File: app/src/main/kotlin/com/ztc1997/fingerprint2sleep/FPQAService.kt [hunk 4]
```diff
@@ -8,6 +8,7 @@ import android.hardware.fingerprint.FingerprintManager
...
+import android.util.Log
```
Comment (Eutiago) [inline app/src/main/kotlin/com/ztc1997/fingerprint2sleep/FPQAService.kt:11]: 11

**Review by Eutiago — APPROVED** [review]

**Pull request closed without merging**